locale/de/LC_MESSAGES/pronterface.po

```diff
@@ -1390,773 +1286,1277 @@ msgstr ""
 
-#: printrun/pronterface.py:703
+#: printrun/pronterface.py:867
 msgid "<&New...>"
 msgstr "<&Neu...>"
 
-#: printrun/pronterface.py:704
+#: printrun/pronterface.py:868
 msgid " Options dialog"
 msgstr " Optionen Dialog"
 
-#: printrun/pronterface.py:704
+#: printrun/pronterface.py:868
 msgid "&Options"
 msgstr "&Optionen"
 
-#: printrun/pronterface.py:706
+#: printrun/pronterface.py:870
 msgid " Adjust slicing settings"
 msgstr " Slicing Einstellungen anpassen"
 
-#: printrun/pronterface.py:706
-#, fuzzy
+#: printrun/pronterface.py:870
 msgid "Slicing settings"
 msgstr "Slicing Einstellungen"
 
-#: printrun/pronterface.py:708
+#: printrun/pronterface.py:872
 msgid "Debug communications"
-msgstr ""
+msgstr "Debugge die Kommunikationen"
 
-#: printrun/pronterface.py:709
+#: printrun/pronterface.py:873
 msgid "Print all G-code sent to and received from the printer."
 msgstr ""
 
-#: printrun/pronterface.py:713
+#: printrun/pronterface.py:877
+msgid "Don't autoscroll"
+msgstr ""
+
+#: printrun/pronterface.py:878
+msgid "Disables automatic scrolling of the console when new text is added"
+msgstr ""
+
+#: printrun/pronterface.py:882
 msgid "&Settings"
 msgstr "&Einstellungen"
 
-#: printrun/pronterface.py:719
-#, fuzzy
+#: printrun/pronterface.py:888
 msgid "&About Printrun"
-msgstr "Drucker überwachen"
+msgstr "&Über Printrun"
 
-#: printrun/pronterface.py:719
+#: printrun/pronterface.py:888
 msgid "Show about dialog"
-msgstr ""
+msgstr "Über Dialog anzeigen"
 
-#: printrun/pronterface.py:720
+#: printrun/pronterface.py:889
 msgid "&Help"
-msgstr ""
+msgstr "&Hilfe"
 
-#: printrun/pronterface.py:746
+#: printrun/pronterface.py:920
 msgid ""
 "Printrun is a pure Python 3D printing (and other types of CNC) host software."
 msgstr ""
+"Printrun ist eine reine Python Hostsoftware für 3D-Druck (und andere Arten "
+"von CNC)."
 
-#: printrun/pronterface.py:750
-#, fuzzy
+#: printrun/pronterface.py:924
 msgid "%.02fmm of filament have been extruded during prints"
-msgstr "mm Filament in Druck genutzt\n"
+msgstr "%.02fmm Filament in Druck genutzt"
 
-#: printrun/pronterface.py:781
+#: printrun/pronterface.py:955
 #, fuzzy
 msgid "Monitor printer status"
 msgstr "Drucker überwachen"
 
-#: printrun/pronterface.py:781
+#: printrun/pronterface.py:955
 msgid ""
 "Regularly monitor printer temperatures (required to have functional "
 "temperature graph or gauges)"
 msgstr ""
 
-#: printrun/pronterface.py:782
+#: printrun/pronterface.py:956
 msgid "Path to the simarrange binary to use in the STL plater"
 msgstr ""
 
-#: printrun/pronterface.py:782
+#: printrun/pronterface.py:956
 msgid "Simarrange command"
 msgstr ""
 
-#: printrun/pronterface.py:783
+#: printrun/pronterface.py:957
 msgid "Circular build platform"
 msgstr ""
 
-#: printrun/pronterface.py:783
+#: printrun/pronterface.py:957
 msgid "Draw a circular (or oval) build platform instead of a rectangular one"
 msgstr ""
+"Zeichnen Sie eine kreisförmige (oder ovale) Plattform anstelle einer "
```

Comment:
This should be "Zeichnet" instead of "Zeichnen Sie" - it describes what the program does, not what the user is meant to do